server/app/unit_test/README.md

```diff
@@ -0,0 +1,14 @@
+## Unit test
+
+The unit tests can be run with the command:
+```
+python -m unittest test_cases
+```
+
+It will take a few seconds to run the test cases, you will see the following message if all test cases pass:
+```
+----------------------------------------------------------------------
+Ran 8 tests in 1.967s
+
+OK
+```
```

Comment:
When it runs, can it give you some output messages of the tests it passed?

Why didn't we use django.tests?